drivers/overlay/joinleave.go

```diff
@@ -224,6 +224,7 @@ func (d *driver) Leave(nid, eid string) error {
 		return types.InternalMaskableErrorf("could not find endpoint with id %s", eid)
 	}
 
+	logrus.Errorf("The channel is valid:%t", d.notifyCh != nil)
```

Comment:
This error message is a bit odd "the channel is valid"; should this be "invalid"? Also missing a space after `:`.

Consider using just `Error` instead of `Errorf`, because printing booleans doesn't need formatting;

```go
logrus.Error("The channel is valid: ", d.notifyCh != nil)
```

Reply:
sorry my bad, this is an error logs that I put for debug, I will remove it